fe/fe-core/src/main/java/org/apache/doris/statistics/ColumnStatistic.java

```diff
@@ -110,6 +110,8 @@ public static ColumnStatistic fromResultRow(ResultRow resultRow) {
             columnStatisticBuilder.setNumNulls(Double.parseDouble(resultRow.getColumnValue("null_count")));
             columnStatisticBuilder.setDataSize(Double
                     .parseDouble(resultRow.getColumnValue("data_size_in_bytes")));
+            columnStatisticBuilder.setAvgSizeByte(columnStatisticBuilder.getDataSize()
```

Comment:
if there are some rows to be deleted or rows need to be merged, the AvgSizeByte may becomes larger, is that right?